content/api.md

```diff
@@ -4,15 +4,150 @@ title: API usage
+
+The public key used for signing is published as a CERT record on `security.2fa.directory`. In the event that the GitHub repository, domain name, or build process is compromised, the key will be missing or revoked.
+
+## FAQ
+
+### How often is the data updated?
+
+Changes to the data are reflected within a minute of being made on the GitHub repository. The frequency of changes in the GitHub repository varies from several per day to a few per week.
+
+### How do I verify the PGP signatures?
+
+Using GPG, you can download the public key of our signing key.
+
+```bash
+gpg --auto-key-locate cert --locate-keys security@2fa.directory
+```
+
+You can then verify a `.json.sig`-file using:
+
+```bash
+gpg --verify <file>.sig <file>
+```
+
+## Version 4 {#v4}
+
+Version 4 of the 2FA Directory JSON files focuses on only outputting data relevant to third-party applications.
+We have removed elements such as service names, icon paths, and service categories.
+
+### URIs
+
+The following JSON files are available for Version 4, each serving different 2FA methods and data requirements.
+
+| Coverage                    | Unsigned File                                                                | PGP Signed File                                                                      |
+|-----------------------------|------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
+| All sites                   | [v4/all.json](https://api.2fa.directory/v4/all.json)                         | [v4/all.json.sig](https://api.2fa.directory/v4/all.json.sig)                         |
+| SMS 2FA                     | [v4/sms.json](https://api.2fa.directory/v4/sms.json)                         | [v4/sms.json.sig](https://api.2fa.directory/v4/sms.json.sig)                         |
+| Email 2FA                   | [v4/email.json](https://api.2fa.directory/v4/email.json)                     | [v4/email.json.sig](https://api.2fa.directory/v4/email.json.sig)                     |
+| non-U2F hardware 2FA tokens | [v4/custom-hardware.json](https://api.2fa.directory/v4/custom-hardware.json) | [v4/custom-hardware.json.sig](https://api.2fa.directory/v4/custom-hardware.json.sig) |
+| U2F hardware tokens         | [v4/u2f.json](https://api.2fa.directory/v4/u2f.json)                         | [v4/u2f.json.sig](https://api.2fa.directory/v4/u2f.json.sig)                         |
+| TOTP (RFC-6238)             | [v4/totp.json](https://api.2fa.directory/v4/totp.json)                       | [v4/totp.json.sig](https://api.2fa.directory/v4/totp.json.sig)                       |
+| non-RFC-6238 software 2FA   | [v4/custom-software.json](https://api.2fa.directory/v4/custom-software.json) | [v4/custom-software.json.sig](https://api.2fa.directory/v4/custom-software.json.sig) |
+
+### Elements
```

Comment:
Should we mention that some keys may be absent?

Reply:
Sure! How would you word it?

Reply:
How about "Note: only the `methods` array is guaranteed to be present."?